services/submission.js

```diff
@@ -1665,13 +1668,17 @@ class Submission {
             if (!newSubmitter) {
                 throw new Error(replaceErrorString(ERROR.INVALID_SUBMISSION_NO_SUBMITTER, submitterID));
             }
-            if (newSubmitter?.userStatus === USER.STATUSES.INACTIVE || newSubmitter?.role !== ROLES.SUBMITTER) {
+            if (newSubmitter?.userStatus === USER.STATUSES.INACTIVE) {
```

Comment:
We need to check if newSubmitter has "data_submission:create" permission, also has access to study of current submission.